m4_sessions/1710-inning/group_four.md

```diff
@@ -1,31 +1,80 @@
+
+**Session Outline:**
+
+**1. Intro** (5 minutes)
+
+Quick intros of presenters, participants, and the topic for the hour
+
+**2. Inform:** (20 minutes)
+
+Examples of social movements and the role of technology 
+
+_Present for ~5 minutes on each of the below social movements and how it helps inform our commitments to the greater good of our communities_
 
+- [Coders for Bernie](https://coders.forsanders.com/) (Daniel)
+  - What is election tech and why is it important
+  - Description of the politically related open source projects for Bernie Movement
+  - Daniel's Capstone project
+  
+- [Maker Movement](https://askatechteacher.com/2015/06/04/the-maker-movement-in-education/) in Ed (Michael)
+  - Transition from lecture style instruction to a more hands-on/participatory style of instruction
+  - Capstone project idea: tracking progress/school-based metric data reporting technology
+  
+- [North Korea](https://www.libertyinnorthkorea.org/) (Jonathan)
+  - Balloon drops w/ thumb drives of S. Korean and other media - opening the curtains to the world
+  - Organization tracking container ships via saletite (BYOB project looking to build an API of the data)
+  - Impact of radio on helping people make the decision to defect
+  - People in NK and taking pictures to raise awareness and make it real
+  - LINK - Liberty In North Korea
+  - Jonathan is still in the exploration phase and working on how he can be involved and make an impact
+  
+- [Code For Social Good](https://app.code4socialgood.org/) (David)
+  - We don't all need to be working towards a specific movement in the way we help
+  - Code For Good provides access to a list of projects that can add to the collective good
+  - Example project that David wants to start working on after Turing
+  - Quick go-around: what projects jumped out to you when looking through _Code For Social Good_
 
-Session Outcomes: What will students leave with or gain from this session?
+**3. Activate:** (25 minutes)
 
+Individual Exercise within Groups 
 
-Session Outline:  Include the flow of the session and the time for each section.
+_We'll split the room into 5 groups and each of the presenters will sit with a group to push the conversation forward._
 
-**Intro**
+- (10 minutes) Think about a movement you are passionate about and think about ways you can be involved as a developer - the distinction being that we are focused on how we can use our skills to further the development of technology that helps social movements for good (not use technology)
+  - ****Worksheet****
+    - Who/what are the sides? What are their perpsectives/motivations?
+    - What is being tried/has already been tried? 
+    - How can technology help push the movement forward?
 
-- Conversation
-- Diary
-- Examples of social movements and the role of technology.
-- Design thinking.
+- (5 minutes) Write down some committments you want to make to being an active voice in/contributor to the movement/s you care about
```

Comment:
could be cool to have them come back together and share their commitments on the white board or in the slack channel so that people can follow up after the session

Reply:
@emhickmann I like it!  What about having someone write it on poster paper to hang in the hallway for the rest of the Mod?

Reply:
@sljohnson32 You can grab chart paper from me if you'd like to do this!

Reply:
I'm totally cool with this idea!